src/Resources/templates/parameters.php.twig

```diff
@@ -1,9 +1,18 @@
-{% for parameter in parameters %}
-{{ type_hinter.typeHint(parameter) -}}
-{# TODO add typehint for class. Cant call parameter.class because that will trigger the autoloader #}
-${{  parameter.name }}
-{%- if parameter.isOptional() %}
-{%- if parameter.allowsNull() %} = null{% else %} = {{ parameter.defaultValue }}{% endif %}
-{%- endif %}
-{% if not loop.last %}, {% endif %}
+{% for parameter in parameters -%}
+    {% if parameter.hasType -%}
+        {% if (not parameter.isOptional() and parameter.allowsNull())
```

Comment:
Really complicated if statement for a template. Can we add a function on parameter, so it's easier to see the intent behind this code?